drv/transceivers-server/src/main.rs

```diff
@@ -657,7 +658,7 @@ fn main() -> ! {
         #[derive(Copy, Clone, Enum)]
         enum Timers {
             I2C,
-            SPI,
+            Spi,
```

Comment:
I'd vote for a clippy exception here, since having `I2C` and `Spi` in the same enum is awkward.